src/Pages/Wallets.razor

```diff
@@ -42,45 +42,40 @@
                         <Button Color="Color.Success" TextColor="TextColor.Light" Clicked="@context.Clicked" Block>New</Button>
                     </NewCommandTemplate>
                     <EditCommandTemplate>
-                        <Button Color="Color.Primary" Clicked="@context.Clicked" Block>Edit</Button>
+                        <Button Color="Color.Primary" Clicked="@context.Clicked" @onclick:stopPropagation="true" Block>Edit</Button>
```

Comment:
Using @events:action with Blazorise has given me lots of issues in the past 🤔

Reply:
this prevents the bug of the wallet keys displaying when you click a button. I can remove this if this causes problems, since is not a super annoying bug

Reply:
I would say maintain it if it works 👯